README.md

```diff
@@ -318,7 +334,8 @@ If your object is an instance of `UIViewController` class, perform the next step
 	[One addInteractionResponseDelegate:<your-object>];
 	```
 
-*Note:* the SDK will weakly store your object, so you need to keep a strong reference to it somewhere. 
+*Note:*
+- The SDK will weakly store your object, so you need to keep a strong reference to it somewhere.
```

Comment:
store? or the SDK has a weak reference to your object? "Weakly store" to me sounds like an incorrect wording

Reply:
@aherod20 we use https://developer.apple.com/documentation/foundation/nsmutablearray/1411274-addobject